content/md/assets/family-by-family-pam-migration.md

```diff
@@ -0,0 +1,104 @@
+---
+id: family-by-family-pam-migration
+title: The **family-by-family** migration guide for PAM assets
+popular: false
+ee-only: true
+related: from-PAM-to-new-AM, how-to-migrate-from-pam
+---
+
+# What's the family-by-family migration?
+In this migration, you can have **several asset families** for your existing PAM assets.
+
+You will need to define manually 2 things:
+- For each asset, its asset family.
+- For each asset collection product attribute, the asset family linked.
```

Comment:
Shouldn't they also have to get a map of all the product attributes linked to the created asset families ?